.github/workflows/2_auto_publish_release.yml

```
@@ -29,8 +29,7 @@ jobs:
     - name: Setup Python
       uses: actions/setup-python@v4
       with:
-        # return to 3.x once compatible with 3.10+ (pyuv)
-        python-version: '3.9'
+        python-version: '3.11'
```

3.11 here but 3.x in stage 1?

---

Not sure why I did that?

```suggestion
        python-version: '3.x'
```